src/htmlContent/update-dialog.html

```diff
@@ -1,17 +1,17 @@
 <div class="update-dialog modal">
     <div class="modal-header">
+        <img class="update-icon" src="styles/images/update_large_icon.svg">
```

Comment:
Check the validity of the changed HTML on `linux`.

Reply:
@ayushrathi15 , please validate this in the linux builds.